README.md

```diff
@@ -38,16 +46,51 @@ To edit this project:
+- Twitter: [![Twitter Badge](https://img.shields.io/badge/-Pancho2788-white?logo=Twitter&logoColor=1DA1F2&style=plastic)](https://twitter.com/Pancho2788)
 
-- LinkedIn: [Francisco Borja](https://www.linkedin.com/in/francisco-borja-lobato/)
+- LinkedIn: [![LinkedIn Badge](https://img.shields.io/badge/-FranciscoBorja-white?logo=LinkedIn&logoColor=1DA1F2&style=plastic)](https://www.linkedin.com/in/francisco-borja-lobato/)
 
+👤 **Fabien**
+
+- GitHub: [![GitHub Badge](https://img.shields.io/badge/-fibini-white?logo=GitHub&logoColor=181717&style=plastic)](https://github.com/fibini)
+
+- Twitter: [![Twitter Badge](https://img.shields.io/badge/-Onenewpage1-white?logo=Twitter&logoColor=1DA1F2&style=plastic)](https://twitter.com/Onenewpage1)
+
+- LinkedIn: [![LinkedIn Badge](https://img.shields.io/badge/-FabienBrathwaite-white?logo=LinkedIn&logoColor=1DA1F2&style=plastic)](https://www.linkedin.com/in/fabien-brathwaite/)
+
+👤 **Tinashe Timothy Murambinda**
+
+- GitHub: [![GitHub Badge](https://img.shields.io/badge/-Draxeytina-white?logo=GitHub&logoColor=181717&style=plastic)](https://github.com/Draxeytina/)
+- Twitter: [![Twitter Badge](https://img.shields.io/badge/-tinamura2-white?logo=Twitter&logoColor=1DA1F2&style=plastic)](https://twitter.com/tinamura2")
+- LinkedIn: [![LinkedIn Badge](https://img.shields.io/badge/-TinasheMurambinda-white?logo=LinkedIn&logoColor=1DA1F2&style=plastic)](https://www.linkedin.com/in/timothy-tinashe-murambinda/")
+
+👤 **Waqas Ahmad**
+
+- GitHub: [![GitHub Badge](https://img.shields.io/badge/-waqaskanju-white?logo=GitHub&logoColor=181717&style=plastic)](https://github.com/waqaskanju)
+- Twitter: [![Twitter Badge](https://img.shields.io/badge/-waqaskanju1-white?logo=Twitter&logoColor=1DA1F2&style=plastic)](https://twitter.com/waqaskanju)
+- LinkedIn: [![LinkedIn Badge](https://img.shields.io/badge/-WaqasKanju-white?logo=LinkedIn&logoColor=1DA1F2&style=plastic)](https://linkedin.com/in/waqaskanju)
+
+## Acknowledgments
+This project's [design](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign) was inspired by [Murat Korkmaz](https://www.behance.net/muratk) under [Creative Commons License](https://creativecommons.org/licenses/).
+
+## 👤 Waqas Ahmad
+
+- GitHub: [@waqaskanju](https://github.com/waqaskanu)
+
+- Twitter: [@waqaskanju](https://twitter.com/waqaskanju)
+
+- LinkedIn: [Waqas Kanju](https://www.linkedin.com/in/waqaskanju/)
+
+## Acknowledgement
+  This project's [design](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign) was inspired by [Murat Korkmaz](https://www.behance.net/muratk) under [Creative Commons License](https://creativecommons.org/licenses/).
+
+  
 ## :star2: Show your support 
 Give a :star: if you like my project! 
 
 ## :pencil: License
-This project is [MIT](https://github.com/Gopxfs/Back-end_Tesla-Booking/blob/main/LICENSE) licensed. 
+This project is [MIT](https://github.com/Draxeytina/Front-end_Tesla-Booking/blob/main/LICENSE) licensed. 
```

Comment:
- [ ] On the "license" section, the link is broken because it is pointing to a file does not exist.  Please, kindly go [here](https://github.com/microverseinc/readme-template) and grab the `MIT.md` file, copy it to the root of your project, and link to it on your readme's license section.

That way, your app will be well documented 💪

Reply:
Done ✔️